src/Util/Casacore.cc

```diff
@@ -12,6 +12,18 @@
 #include "ImageData/CartaMiriadImage.h"
 #include "Logger/Logger.h"
 
+/**
+ * @details This function checks and resolves the given top-level and starting directories.
+ * If default placeholder values ("base" or "root") are found, they are replaced accordingly.
+ * The function verifies that both directories exist and are accessible, and ensures that
+ * the starting directory is a valid subdirectory of the top-level directory.
+ *
+ * @note If `starting_string` is invalid, it is replaced with `top_level_string`.
+ * @note If `starting_string` is not a subdirectory of `top_level_string`, the function logs a critical error and returns `false`.
+ *
+ * @warning If both `top_level_string` and `starting_string` are set to their default placeholders ("base" and "root"),
+ *          the function logs a critical error and returns `false`.
+ */
 bool CheckFolderPaths(string& top_level_string, string& starting_string) {
```

Comment:
Doxygen isn't correctly matching and merging the docs for this function because it's using bare `string` as a type here (it's `std::string` in the header). Years ago we decided to standardise and use the `std` namespace explicitly everywhere, so this should be fixed -- please add the `std::` to all the bare type names in this file (I had a look and I think it's just `string` in a bunch of places).